ihmc-sensor-processing/src/main/java/us/ihmc/sensorProcessing/sensorProcessors/SensorProcessing.java

```diff
@@ -936,15 +943,61 @@ public void addJointPositionAffineTransformWithJointsToIgnore(DoubleProvider sca
          String prefix = JOINT_POSITION.getProcessorNamePrefix(AFFINE);
          String suffix = JOINT_POSITION.getProcessorNameSuffix(jointName, processors.size());
          YoDouble filteredJointPosition = new YoDouble(prefix + suffix, registry);
-         ProcessingYoVariable processor;
+         OffsettingProcessorVariable processor;
```

Comment:
These can't just be lambda functions anymore, unfortunately, because of the additional returns.